src/main/scala/io/altoo/akka/serialization/kryo/DefaultKryoInitializer.scala

```diff
@@ -17,10 +17,18 @@ class DefaultKryoInitializer {
    * Can be overridden to set a different field serializer before other serializer are initialized.
    * Note: register custom classes/serializer in `postInit`, otherwise default order might break.
    */
-  def preInit(kryo: ScalaKryo): Unit = {
+  def preInit(kryo: ScalaKryo, system: ExtendedActorSystem): Unit = {
     kryo.setDefaultSerializer(classOf[com.esotericsoftware.kryo.serializers.FieldSerializer[_]])
+    preInit(kryo)
   }
 
+  /**
+   * Can be overridden to set a different field serializer before other serializer are initialized.
+   * Note: register custom classes/serializer in `postInit`, otherwise default order might break.
+   */
+  @deprecatedOverriding
```

Comment:
Makes sense, but annotation should describe the deprecation like: 
`class deprecatedOverriding("Use preInit(kryo,system) instead", since: String = "2.0.0")`